### ISSUE TITLE
increased activation of mitotic cell cycle spindle assembly checkpoint parent

### ISSUE THREAD

**Issue by ValWood** [pombase/fypo#3172]
increased activation of mitotic cell cycle spindle assembly checkpoint

is a descendant of 

 abnormal mitotic spindle assembly checkpoint

here, the checkpoint isn't abnormal, its fine, but it activated more than it would be normally.

Maybe we need to say this in a different way.

Its the same issue we come across repeatedly in GO.  

Activation of the checkpoint is not equivalent to "positive regulation of the checkpoint"
(I'm not even sure that the positive regulation terms  should exist?)

These are phenotypes which cause an issue that results in "activation of the checkpoint"

**Comment by mah11** [comment]
1. Would it make sense to merge FYPO:0002638 with either of these?

    id: FYPO:0005726
    name: abolished deactivation of mitotic spindle assembly checkpoint

    id: FYPO:0005727
    name: decreased rate of deactivation of mitotic spindle assembly checkpoint

Or should FYPO:0002638 go obsolete and suggest FYPO:0005726 or FYPO:0005727?

2. Whether we merge or not, what should the parent(s) be? FYPO:0005726 ash FYPO:0005727 are also is_a: FYPO:0000168 right now, so presumably the same concern applies.

**Comment by ValWood** [comment]
nope, they are different.


FYPO:0002638 - increased activation of mitotic cell cycle spindle assembly checkpoint
means that there is a problem causing the checkpoint to be (normally) activated.
It isn't positive regulation though.


id: FYPO:0005726
name: abolished deactivation of mitotic spindle assembly checkpoint

AND
id: FYPO:0005727
name: decreased rate of deactivation of mitotic spindle assembly checkpoint

are both related to real regulation of the spindle checkpoint, (i.e the pathway turning it off  the signalling after the checkpoint is satisfied, and the anaphase promoting complex activation has occurred.


FYPO:0002638 should not go obsolete.

I'm probably not explaining very well.

FYPO:0002638   is used when a mutant causes a problem that activates the checkpoint.  It isn't part of the signalling  or even regulating the signalling, it is often way,way, way upstream (in the splicing machinery for example).
Its the thing causing a problem that is detected by the checkpoint.
So the problem *results in* checkpoint activation, but it isn't regulation.

In GO activation is considered to be a  type of "positive regulation"
but in this case isn't a positive regulation. This is the  thing which causes the enrichment issues for cell cycle  researchers.

If everything which, when mutated  resulted in the activation of a spindle checkpoint were considered to be a positive regulation of the spindle checkpoint hundreds of things (everything elongated for example is activated G2/M checkpoint), would be a positive regulator of the checkpoint, and it isn't.

So I wonder if this is somehow caused by coupling of "activation of the checkpoint" and "positive regulation  of the checkpoint" in GO.

Recently I have been thinking that there should not be  "positive regulation of checkpoint" terms at all. The checkpoint signalling would be defined as beginning with the "sensor" and including the signalling  (i.e it would just be the signalling pathway). I'm not sure that a positive regulation term is required, or if it is, it is the sensor that is doing the positive regulation.

>  FYPO:0005726 and FYPO:0005727 are also is_a: FYPO:0000168 right now, so presumably the same concern applies.

So the parentage of FYPO:0005726 and FYPO:0005727 is fine because these really are abnormal negative regulation

Does this help you? If not  we might have to  think about this more, it has bothered  me for a while with GO (it's probably  the last big issue in the cell cycle checkpoint branch, where things don't seem quite right).

**Comment by mah11** [comment]
Thanks for the additional info. Now I think part of the problem lies with what we define as "normal" and "abnormal" in checkpoint contexts.

One point of view is that normal means everything about a checkpoint is the same as wild type, including whether it's triggered as well as whether it works correctly once triggered. I would then say "abnormal" means that in a mutant it could either go wrong once triggered, or be triggered when it shouldn't, or not be triggered when it should.

But it sounds like you're saying that FYPO:0002638 has been used in cases where a mutation creates a situation that triggers the checkpoint, and the checkpoint then acts as it does in wild type cells.

To me, it would only make sense to call a phenotype _increased_ checkpoint activation if a mutant reacted to a given insult by triggering the checkpoint more that wild type cells subjected to the same conditions. In other words, I think of "increased checkpoint activation" as meaning the checkpoint is hypersensitive or hyperactive, not that it's responding as it should to the increased presence of the problem it detects.

Looking at the text definition again now, I see that it doesn't do a great job of making the point that "increased" has to be relative to wild type.

We also have inconsistent handling of "increased activation" for a couple other terms. FYPO:0005512 'increased activation of meiosis I spindle assembly checkpoint' looks like it was modeled on FYPO:0002638 and therefore has the same problems, but there's also:

    id: FYPO:0001707
    name: increased mitotic DNA damage checkpoint activation
    def: "A cell cycle checkpoint phenotype in which the incidence of mitotic cell cycle arrest or delay due to regulation by any mitotic DNA damage checkpoint is greater than in wild type under any specified set of conditions."

The FYPO:0001707 logical def simply says "increased GO:0044773", which is the GO BP for the checkpoint itself, not a regulation term.

I am all the more tempted to make all three of these terms obsolete because of the def issues - two of them have had logical definitions and text definitions that disagree with each other, quite apart from whether GO should keep or ditch "positive regulation of x checkpoint" terms. If you are _absolutely certain_ that they have only been used to mean "more of the problem", never to mean "mean more checkpoint signaling and response than wild type for a given level of problem", then we can rejig with the existing IDs.

But ... I'm still not sure how to capture what you actually want - "mutation causes problem, and checkpoint detects problem, so checkpoint is active". IOW, I'm not sure what to change them to (apart from removing the kps). I really don't want to use "increased activation" in the name because of what I said above - for everything else, increased process x means problem with process x or its regulation.

Checkpoints just don't play nice with the way GO models processes for everything else, with equally frustrating knock-on effects on FYPO.

**Comment by ValWood** [comment]
Yes you have hit the nail on the head. I knew of the problem  for a while (more from people annotating such phenotypes to the checkpoint in GO) but did not know how to describe  the issue, or how you would fix it ....

I know that I have always used these terms  when  the mutant is
responding as it should to the increased presence of the problem it detects.

I guess it is really a readout phenotype. 
Compared to WT the checkpoint is much more active, because in WT the  problems caused by the mutant are absent.

In papers the authors usually refer to this as "checkpoint signalling is increased" or "activated" which doesn't help....

what we really want to say is "normal checkpoint signalling in response to defect caused by mutant"

It is important  because the problem can be pinpointed as being upstream/within/ or downstream of the checkpoint pathway, and can separate problems with signalling from physical problems that can not be fixed.

We could just say  "normal checkpoint signalling"  but that always seemed strange to me too because the signalling would not be  on in the WT cell.

**Comment by ValWood** [comment]
Just to complicate more (or help, I'm not sure now).

Maybe it should be  "increased duration of checkpoint signalling" in these cases. This might work for the spindle checkpoint, because the spindle checkpoint would be active until satisfied  i.e  correct tension achieved. So in this case more signalling seems to just mean, there is a problem, so the signal is active for longer.

I'm not sure about DNA integrity checkpoints  though, if they are on by default until replication is complete, or if they really are only switched on when a problem is detected. I have asked about this but I never found out (I don't think it is known?)

**Comment by mah11** [comment]
on the walk home I had another thought, about the GO angle ...

I'd been struggling to reconcile your querying the positive regulation of checkpoint terms with the fact that turning some signaling on does fit GO's general definition of positive regulation.

Then I realized that it's not that activation of whatever checkpoint isn't a thing that happens, or even that turning checkpoint signaling on isn't positive regulation - it's that it's normally _not a thing gene products do_. Whatever causes the detected problem also causes the checkpoint activation, but gene products don't do that when they're working properly; they only activate checkpoints when they're broken or missing.

**Comment by mah11** [comment]
I like the "increased duration of checkpoint signaling" idea generally. Maybe it could even work no matter whether the un-triggered duration is short (as with the spindle checkpoint) or as long as S phase itself ... ?

**Comment by ValWood** [comment]
> Whatever causes the detected problem also causes the checkpoint activation, but gene products don't do that when they're working properly; they only activate checkpoints when they're broken or missing.

Yes! The first thing that is part of the checkpoint is the sensor.
